.github/workflows/release-master.yaml

```diff
@@ -72,6 +72,24 @@ jobs:
           path: /tmp/digests/*
           if-no-files-found: error
           retention-days: 1
+  helm:
+    permissions:
+      contents: read
+      packages: write
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install Helm
+        uses: azure/setup-helm@v3
+        with:
+          token: ${{ secrets.GH_PAT }}
+      
+      - name: Build and Push the Helm Charts to GitHub Container Registry
+        uses: JimCronqvist/action-helm-chart-repo@master
```

Comment:
Is it directly based on the image version to determine the chart version?

Reply:
The chart version is determined with the version specified in Chart.yml (deploy/charts/kmesh-helm/Chart.yaml)

For now, it has to be updated manually but we could add a script to github action to update accordingly to our needs.

Reply:
we have `make helm-package` and `make helm-push`, which can be used 

```
helm-package.%: # Package Helm chart
	$(eval COMMAND := $(word 1,$(subst ., ,$*)))
	$(eval CHART_NAME := $(COMMAND))
	helm lint $(CHARTS_FOLDER)/${CHART_NAME}
	helm package $(CHARTS_FOLDER)/${CHART_NAME} --app-version ${VERSION} --version ${CHART_VERSION} --destination ${CHART_OUTPUT_DIR}/

.PHONY: helm-push.%
helm-push.%: helm-package.%
	$(eval COMMAND := $(word 1,$(subst ., ,$*)))
	$(eval CHART_NAME := $(COMMAND))
	helm push ${CHART_OUTPUT_DIR}/${CHART_NAME}-${CHART_VERSION}.tgz ${CHART_OCI_REGISTRY}

.PHONY: helm-package
helm-package:
	@for chart in $(CHARTS); do \
      	$(MAKE) $(addprefix helm-package., $$(basename $${chart})); \
	done

.PHONY: helm-push
helm-push: helm-package
	# run other make targets
	@for chart in $(CHARTS); do \
	  	$(MAKE) $(addprefix helm-push., $$(basename $${chart})); \
	done
```